src/ui/app-pages/WidgetEditPage.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import InternalPage from 'ui/internal-page/InternalPage';
+import { Breadcrumb, OverlayTrigger, Popover, Grid } from 'patternfly-react';
+import { BreadcrumbItem } from 'frontend-common-components';
+import { FormattedMessage } from 'react-intl';
+import WidgetEditFormContainer from 'ui/widgets/WidgetEditFormContainer';
+
+const WIDGET_EDIT_HELP = 'widget.page.edit.help';
+
+const popover = () => (
+  <Popover id="popover-admin-app-switch" title="">
+    <p>
+      <FormattedMessage id={WIDGET_EDIT_HELP} />
+    </p>
+  </Popover>
+);
+
+const WidgetEditPage = () => (
+  <InternalPage className="WidgetEditPage">
+    <Grid fluid>
+      <Breadcrumb>
+        <BreadcrumbItem route="home">
+          <FormattedMessage id="menu.uxPattern" />
+        </BreadcrumbItem>
+        <BreadcrumbItem route="home">
```

Comment:
Here should be the Widget list page route. Since it's not released yet, we'll leave it as is but please set a comment like `{/* FIXME replace route with widget list route when available */}